engine-tests/src/main/java/org/terasology/Environment.java

```diff
@@ -34,12 +36,13 @@
 
     private static final Logger logger = LoggerFactory.getLogger(Environment.class);
 
+    protected Context context;
```

Comment:
Given their is a public getContext() method I would suggest making this private, unless subclasses are replacing the context?